docs/reference/query-dsl/geo-queries.asciidoc

```diff
@@ -9,19 +9,22 @@ lines, circles, polygons, multi-polygons, etc.
 The queries in this group are:
 
 <<query-dsl-geo-bounding-box-query,`geo_bounding_box`>> query::
-Finds documents with geopoints that fall into the specified rectangle.
+Finds documents with geoshapes or geopoints which intersect the specified rectangle.
 
 <<query-dsl-geo-distance-query,`geo_distance`>> query::
-Finds documents with geopoints within the specified distance of a central point.
+Finds documents with geoshapes or geopoints within the specified distance of a central point.
 
 <<query-dsl-geo-polygon-query,`geo_polygon`>> query::
-Find documents with geopoints within the specified polygon.
+Find documents with geoshapes or geopoints which intersect the specified polygon.
 
 <<query-dsl-geo-shape-query,`geo_shape`>> query::
+Finds documents with geoshapes or geopoints which either intersect, are contained by, or do not intersect
```

Comment:
Should end with `the specified geoshape`

Reply:
```suggestion
Finds documents with geoshapes or geopoints which either intersect, are contained by, or do not intersect the specified geoshape.
```

Reply:
What is meant by `or do not intersect`? That sounds like a mistake, but was in the original text.